tools/ci/download_bin.sh

```diff
@@ -66,6 +57,32 @@ download_artifact()
 
 }
 
-parse_params $@
-download_artifact
+download_artifact_macOS(){
+    echo "unsupported for now."
+    exit 1
+    # TODO: https://developer.github.com/v3/actions/artifacts/#download-an-artifact
+    local workflow_artifacts_url="$(curl https://api.github.com/repos/FISCO-BCOS/FISCO-BCOS/actions/runs\?branch\=${branch}\&status\=success\&event\=push | grep artifacts_url | head -n 1 | cut -d \" -f 4)"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5514420)